src/tallies/tally_scoring.cpp

```diff
@@ -55,8 +55,8 @@ FilterBinIter::FilterBinIter(const Tally& tally, const Particle* p)
   this->compute_index_weight();
 }
 
-FilterBinIter::FilterBinIter(const Tally& tally, bool end)
-  : tally_{tally}
+FilterBinIter::FilterBinIter(const Tally& tally, bool end, std::vector<FilterMatch>* particle_filter_matches)
```

Comment:
This line is getting a little long. Can you bring that last argument down to a new line?